minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/configuration/notifiers/RestChangeNotifier.java

```diff
@@ -227,15 +235,27 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
                     }
                 }
                 setConfigFile(configBuilder.substring(0,configBuilder.length()-1));
-                notifyListeners();
-                writeOutput(response, POST_TEXT, 200);
+                ListenerHandleResult[] listenerHandleResults = notifyListeners();
+
+                writeOutput(response, getPostText(listenerHandleResults), 200);
```

Comment:
With the new collection of results, it seems like the returned status code should reflect the status of the attempted notifications.